model/configs/default_pcfg.py

```diff
@@ -61,7 +61,17 @@ def get_config():
   # Whether to use annotated number of operations to limit number of loops.
   config.use_annotations = True
   # Extra loops in addition to annotations.
-  config.extra_loops = 2
+  config.extra_loops = 0
+  # Whether to copy input to prediction
+  config.copy_input = False
+  config.copy_input_in_full = False
+  # Whether to copy previous output
+  config.copy_output = False
+
+  config.end_token = None
+  config.in_out_token = None
+  config.sep_token = None
+  config.end_iter_token = 'END'
```

Comment:
Just confirming this is indented, and it's not [END] (with square brackets) or [ENDITER] as above.

Reply:
Yes!